lexicon.md

```diff
@@ -0,0 +1,110 @@
+# Open Source Project Security Lexicon
+
+This lexicon is intended to provide clear and consistent definitions for terms used in the Open Source Project Security Baseline. These definitions help ensure that criteria are interpreted accurately and consistently across projects.
+
+## Definitions
+
+### Arbitrary Code
+
+Code provided by an external source that is executed by a system without validation or restriction.
+
+### Build and Release Pipeline
+
+A series of automated processes that compile and deploy software. Similar to the generic term [CI/CD Pipelines](#cicd-pipelines), but this term excludes some pipelines, such as pre-merge status checks.
+
+### Changes
+
+Any alteration of the project's [codebase](#codebase), [CI/CD Pipelines](#cicd-pipelines), or [documentation](#project-documentation). This may include addition, deletion, or modification of content.
+
+### CI/CD Pipelines
+
+Automated pipelines for Continuous Integration and Continuous Delivery. Responsible for building, testing, and delivering [changes](#changes). These pipelines integrate contributions frequently, enabling rapid and reliable software delivery. CI focuses on testing and building code, while CD delivers software to users.
+
+In the context of the Open Source Project Security Baseline, CI/CD Pipelines does not involve Continuous Deployment.
```

Comment:
Is this document intended to be a lexicon for things outside of baseline? If so, I'm okay with this sentence but we should probably find another home for the lexicon doc long term. If not, it would be simpler to strike *CD* everywhere it appears and not have to explain that in this context CI/CD doesn't mean CD.

Reply:
Next time I'm going to drink more coffee before I start reviewing. 😄 I completely missed the distinction between "delivery" and "deployment" here. If I were to make a suggestion for rephrasing, it would be something like:

> In the context of the Open Source Project Security Baseline, *CD* refers only to *continuous delivery*, not to *continuous deployment*, as sometimes used elsewhere.

Reply:
```suggestion
In the context of the Open Source Project Security Baseline, CD refers only to _continuous delivery_, not to _continuous deployment_, as sometimes used elsewhere.
```